deploy/build-deploy-common.sh

```diff
@@ -0,0 +1,224 @@
+#!/usr/bin/env bash
```

Comment:
Noticed a difference in the shebang.

Reply:
oh yeh - I can force it to be /bin/bash or let env to get the "right Bash" - I think this one is more extended